### PR TITLE
New link for resetting your own password on the RPA Manager login pag…

### PR DESCRIPTION
**Pull request by sebermuley**
…e -> ready for SME review

### CODE REVIEW

**Review by oveltan — DISMISSED** [review]
OK

File: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc [hunk 1]
```diff
@@ -84,6 +84,32 @@ Change your own password regularly. You will be logged out in the process and ha
 . Click *OK*.
 . Log in with your new password.
 
+[[iforgotmypassword]]
+== Reset Your Password
+
+If you forgot your password or if your one-time password has expired, reset it by clicking the link on the RPA Manager login page. You need to know your user name for resetting your password.
```
Comment (IsaacEldridge) [inline rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc:90]:
```suggestion
If you forget your password or if your one-time password expires, click the link on the RPA Manager login page to reset it. You need to know your user name to reset your password.
```

File: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc [hunk 2]
```diff
@@ -84,6 +84,32 @@ Change your own password regularly. You will be logged out in the process and ha
...
+
+. Navigate to the RPA Manager login screen.
+. Click the *Forgot your credentials?* link.
```
Comment (IsaacEldridge) [inline rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc:93]:
```suggestion
. Click *Forgot your credentials?*.
```

**Review by IsaacEldridge — APPROVED** [review]

File: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc [hunk 3]
```diff
@@ -84,6 +84,33 @@ Change your own password regularly. You will be logged out in the process and ha
...
+. In the *Forgot your credentials* form, enter your username.
+. Click *Continue*.
+
+A confirmation request for creating a new password is emailed to you.
+
+. Open the email in your email application.
+. Click *Request New Credentials*.
+
+A link to create a new password is emailed to you. The one-time password expires after 24 hours.
+If you use Two-Factor Authentication, append the time-based one-time password to your one-time password.
```
Comment (Crispy-Salesforce) [inline rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc:103]: I don't think you need to use capital letters for ˝two-factor authentication"

**Review by Crispy-Salesforce — APPROVED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

File: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc (hunk 3 above)
Comment (sebermuley) [reply rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc]: changed

**Comment by sebermuley** [comment]
RPA Manager was released on Thursday.

**Pull request merged**